R/conditions.R

```diff
@@ -384,3 +404,25 @@ notEqualError <- function(names, val, call = sys.call(-1), msg = "", ...) {
   )
   stop(c)
 }
+
+### Added by KSG 6/13/22
+
+#' Variables not equal to value error.
+#'
+#' @param var Names of the variables.
+#' @param val Value to be equal to.
+#' @param call sys.call to pass on to the error.
+#' @param msg Additional information for the error message as
+#' glueCollapse'able string.
+#' @noRd
+notEqualError <- function(names, val, call = sys.call(-1), msg = "", ...) {
+  message <- glueCollapse(
+    "{ names *} not equal to {val}!", msg
+  )
+  
+  c <- condition(
+    c("simstudy::notEqual", "error"),
+    message, call, ...
+  )
+  stop(c)
+}
```

Comment:
You seem to have duplicated this whole function?

Reply:
Yes, I did. Fixed.